src/table/components/TableWrapper.jsx

```diff
@@ -113,33 +113,51 @@ export default function TableWrapper(props) {
     });
   }, [rows.length, size.qcy, size.qcx, page]);
 
-  const tableBackgroundColor = theme.getStyle('object', 'straightTable', 'backgroundColor');
+  const SelectProps = {
+    inputProps: {
+      'aria-label': translator.get('SNTable.Pagination.RowsPerPage'),
+      'data-testid': 'select',
+      tabIndex: !keyboard.enabled || keyboard.active ? 0 : -1,
+    },
+    native: true,
+  };
+
   const paperStyle = {
-    border: '1px solid #D9D9D9',
+    border: theme.isBackgroundDarkColor ? '1px solid #F2F2F3' : '1px solid #D9D9D9',
```

Comment:
Do we do this for the old table or some other chart when we change to a dark color?

Reply:
<img width="530" alt="Screenshot 2022-02-22 at 14 24 49" src="https://user-images.githubusercontent.com/4471489/155141318-a9a18d03-7a73-4699-9f1b-4b8ae09a96d3.png">
Yes.

The bottom border is tricky, now it is not possible for me to let it be good in both sense and mashup. I am still finding a way.